stdlib/2and3/pyexpat/__init__.pyi

```diff
@@ -0,0 +1,75 @@
+from typing import List, Tuple, Optional, Callable, Any, Protocol, Union, Dict
+
+import pyexpat.errors as errors
+import pyexpat.model as model
+
+EXPAT_VERSION: str  # undocumented
+version_info: Tuple[int, int, int]  # undocumented
+native_encoding: str  # undocumented
+features: List[Tuple[str, int]]  # undocumented
+
+class ExpatError(Exception):
+    code: int = ...
```

Comment:
These should not have `= ...` because they don't exist on the class. (They also don't exist on instances I construct manually, but presumably they do exist when the module actually raises this error.)